### ISSUE TITLE
tlig/em dashes do not work with certain fonts

### ISSUE THREAD

**Issue by simifilm** [lualatex/luaotfload#332]
Consider the following MWE (with plain `luatex`).

```
 \input luaotfload.sty\relax

    \font\x="file:PalatinoLTStd-Roman.otf:+tlig" at 10pt\relax
    \x abc--xyz

    \bye
```

Although +tlig is set, the two dashes are not rendered as – (em dash).

I've seen this with several fonts, but I could not figure out any systematics behind it. 

I realize that this MWE is not ideal since it relies on a proprietary font, but if needed I can provide the font.

**Comment by phi-gamma** [comment]
···<date: 2016-04-18, Monday>···<from: Simon Spiegel>···

> I realize that this MWE is not ideal since it relies on a
> proprietary font, but if needed I can provide the font.

Without a font it’s completely untestable, sorry. In any case you
should wait for version 2.7 before running Luaotfload with the
pretest.

**Comment by simifilm** [comment]
For what it's worth, this seems to have been fixed with `luaotfload` 2.7. Although there seem to be some new issues here with small caps, but I have to go more into that.

**Comment by phi-gamma** [comment]
Great. Please create a new ticket for that other issue.
